### PR TITLE
Import NavLink  instead of Link

### PR DESCRIPTION
**Pull request by Aadish1233**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -36,3 +36,9 @@
     transform: rotate(360deg);
   }
 }
+nav a.active{
```
Comment (Aadish1233) [inline src/App.css:39]: css added to the navigation tab

**Comment by Aadish1233** [comment]
hey would plz merge the my branch

File: src/components/Navbar.js [hunk 2]
```diff
@@ -49,17 +49,17 @@ export default function Navbar() {
                 <div className="hidden sm:ml-6 sm:block">
                   <div className="flex space-x-4">
                     {navigation.map((item) => (
-                      <Link
+                      <NavLink
                         key={item.name}
                         to={item.href}
                         className={classNames(
-                          item.current ? 'bg-gray-900 text-white' : 'text-gray-300 hover:bg-gray-700 hover:text-white',
```
Comment (TauqeerAhmad5201) [inline src/components/Navbar.js:56]: hey! What are you gaining from this? By removing **bg-gray-900**, it wouldn't be highlighted again.

Comment (Aadish1233) [reply src/components/Navbar.js:56]: If it is not removed. The Home tab is  always highlighted with gray  by default. 
Screenshot (28)

**Comment by Aadish1233** [comment]
hey did u reviewed it

**Comment by TauqeerAhmad5201** [comment]
> hey did u reviewed it

Yes, merged it. Had a code review with the team and it was resolved.

**Pull request merged**